lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1623,6 +1631,11 @@ void StmtEmitter::emitExpression(Value exp,
 
   // Emit each stmt expression in turn.
   for (auto *expr : tooLargeSubExpressions) {
+    // Do not emit assignments for constants, as these will be printed
+    // immediately after declaration of their temporary.
+    if (isConstantExpression(expr) &&
+        tooLargeSubExpressions[0]->getParentOp()->hasTrait<ProceduralRegion>())
```

Comment:
Please remove this change too, it shouldn't be needed if I understand correctly.